OpenSim/Simulation/OpenSense/IMUPlacer.cpp

```diff
@@ -86,7 +86,7 @@ bool IMUPlacer::run(bool visualizeResults) {
 
     _calibrated = false;
     // Check there's a model file specified before trying to open it
-    if (get_model_file().size() == 0) {
+    if (_model.empty() && get_model_file().size() == 0) {
         OPENSIM_THROW(Exception, "No model file specified for IMUPlacer.");
```

Comment:
Could be good to update the exception message to reflect the updated if statement.